TrackerConditions/src/StrawResponse.cc

```diff
@@ -129,19 +129,37 @@ namespace mu2e {
   }
 
   double StrawResponse::halfPropV(StrawId strawId, double kedep) const {
-    return PieceLine(_edep,_halfvp,kedep);
```

Comment:
I can see it was useful to keep the uneven binning code while developing and debugging, but I don't think we will use it again.  Once you've verified the even binning gives equivalent results I suggest to remove all the code for uneven binning (and the eDep array).